Source/BroncoDrome/Runner/Runner.cpp

```diff
@@ -582,6 +582,7 @@ void ARunner::AddToHealth(int newHealth) {
     }
 	
 	if (!this->isAI) {
+	  this->numDeaths ++; // increase the number of deaths
```

Comment:
I believe this will incorrectly increment the deaths. This function that it is nested in, is called whenever health is added or decremented to the runner. Therefore, if the player heals any health, it would add to the number of deaths. I'm not sure if you use this parameter elsewhere, but if you do need it, just move it into the block above anywhere after line 526 where it ensures runner actually died.

Reply:
It doesn't look like you use this later, but if you were having bugs with it, this would be why. If you aren't gonna use it, remove that line

Reply:
Yeah I am actually totally unsure about that because I swear that I removed all references to the "numDeaths" variable. In fact, when I use the search function in my editor, I cannot find the variable in the codebase.

Reply:
switched branches and back and it was there. Fixed now. Good catch.

Reply:
Sweet! All good